integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -2033,4 +2021,61 @@ public boolean accept(File dir, String name) {
     callShellScriptToBuildDeployAppInPod(
         appName, scriptName, username, password, infoDirName, archiveExt);
   }
+
```

Comment:
If this code is not needed, should be removed.

Reply:
This trans doesn't touch that method and doesn't need to use that method either. It's used by TestUtils.java and then BaseTest and then ITMonitoringExporter.java

Reply:
Its showing as you have commented these methods. See the changes in this PR under Files Changed.